### PR TITLE
internal/global,trace: fix spelling error

### PR DESCRIPTION
**Pull request by kevinburkesegment**
There is another one in the generated semconv files but I couldn't find where those were being generated from.

### CODE REVIEW

File: trace/context.go [hunk 1]
```diff
@@ -52,6 +55,7 @@ func SpanFromContext(ctx context.Context) Span {
 	if span, ok := ctx.Value(currentSpanKey).(Span); ok {
 		return span
 	}
+	fmt.Println("returning noop span - span doesn't exist")
```
[the remark this answers is not in the document]
Comment (kevinburkesegment) [reply trace/context.go:58]: Ah, I'm horribly sorry. I added this months ago in the branch I had.

[the remark this answers is not in the document]
Comment (kevinburkesegment) [reply trace/context.go]: This is fixed

File: trace/context.go [hunk 2]
```diff
@@ -14,7 +14,9 @@
 
 package trace // import "go.opentelemetry.io/otel/trace"
 
-import "context"
+import (
+	"context"
+)
```
Comment (dmathieu) [inline trace/context.go:19]: This change is uneeded too.

**Comment by kevinburkesegment** [comment]
I can't add the "Skip Changelog" label but this seems like a change that should not generate a changelog entry.

**Review by dmathieu — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Pull request merged**